julia/test/unittest/ndarray.jl

```diff
@@ -1515,8 +1515,8 @@ function test_argmax()
          4 2 6]
     x = NDArray(A)
 
-    @test copy(argmax(x, dims = 1)) == [2 1 2]
-    @test copy(argmax(x, dims = 2)) == reshape([2, 3], :, 1)
+    @test copy(argmax(x, dims = 1)) == [x[1] for x in argmax(A, dims = 1)]
+    @test copy(argmax(x, dims = 2)) == [x[2] for x in argmax(A, dims = 2)]
```

Comment:
This would be great.
```suggestion
    @test copy(argmax(x, dims = 2)) == [x[2] for x ∈ argmax(A, dims = 2)]
```

Just type `\in<tab>` in Julia REPL

Reply:
I have updated it. Thank you : )